<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhotoChangeClass</class>
 <widget class="QDialog" name="PhotoChangeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>459</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>94</red>
        <green>47</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>94</red>
        <green>47</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>PhotoChange</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pokersphere.qrc">
    <normaloff>:/img/Resources/sphere.ico</normaloff>:/img/Resources/sphere.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="chgtPhotoLabel">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Changement de la photo de l'adh√©rant</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="logoLabel">
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="pokersphere.qrc">:/img/Resources/pokerSphere.gif</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="cameraLabel">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="shootHimLabel">
         <property name="font">
          <font>
           <family>Arial Narrow</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;a href=&quot;a&quot; style=&quot;text-decoration:none;&quot;&gt;&lt;font color=&quot;#5E2F6A&quot;&gt;Shoot Him !&lt;/font&gt;&lt;/a&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="imageLabel">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>240</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="enregistrerLabel">
           <property name="font">
            <font>
             <family>Arial Narrow</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;a href=&quot;a&quot; style=&quot;text-decoration:none;&quot;&gt;&lt;font color=&quot;#5E2F6A&quot;&gt;Enregistrer&lt;/font&gt;&lt;/a&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="annulerLabel">
           <property name="font">
            <font>
             <family>Arial Narrow</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>&lt;a href=&quot;a&quot; style=&quot;text-decoration:none;&quot;&gt;&lt;font color=&quot;#5E2F6A&quot;&gt;Annuler&lt;/font&gt;&lt;/a&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pokersphere.qrc"/>
 </resources>
 <connections/>
</ui>
